client/grpc/client.go

```diff
@@ -119,7 +129,10 @@ func (g *grpcClient) translate(stream drand.Public_PublicRandStreamClient, out c
 func (g *grpcClient) RoundAt(t time.Time) uint64 {
 	ctx, cancel := context.WithTimeout(context.Background(), grpcDefaultTimeout)
 	defer cancel()
-	info, err := g.client.ChainInfo(ctx, &drand.ChainInfoRequest{})
+
+	metadata := common.Metadata{ChainHash: g.chainHash}
```

Comment:
Can we embed that into a function `g.metadata()` so it's shorter and less prone to error for the future ?

Reply:
Done!